src/sass/_patterns-global-nav.scss

```diff
@@ -1,171 +1,307 @@
+      top: auto;
+    }
 
     &__wrapper {
-      box-sizing: border-box;
       background: none repeat scroll 0 0 $global-white;
+      box-sizing: border-box;
       margin: 0 auto;
       position: relative;
       text-align: left;
-      width: 984px;
+      width: 100%;
 
-      @media only screen and (min-width: 768px) {
-        width: auto !important;
+      @media only screen and (min-width: $global-breakpoint-medium) {
+        width: auto;
       }
 
-      @media only screen and (max-width: 768px) {
-        width: 100%;
+      @media only screen and (min-width: $global-breakpoint-large) {
+        width: $global-breakpoint-large;
       }
+    }
+
+    &__title {
+      display: none;
+
+      @media only screen and (max-width: $global-breakpoint-medium) {
+        background-color: transparent;
+        background-image: url('#{$global-asset-path}7bd1bd7b-arrow_down_9fa097.svg');
+        background-position: calc(100% - .625rem) 50%;
+        background-repeat: no-repeat;
+        background-size: .8125rem;
+        border-bottom: 1px solid $global-mid-grey;
+        border-right: 0;
+        border-top: 1px solid $global-mid-grey;
+        color: $global-x-dark-grey;
+        cursor: pointer;
+        display: block;
+        font-family: Ubuntu, Arial, 'libra sans', sans-serif;
+        font-size: .8125rem;
+        font-weight: 300;
+        line-height: 1.5;
+        margin: 0;
+        padding: .625rem 0 .625rem $navigaiton-height;
+        position: relative;
+        text-align: left;
+        width: 100%;
+
+        &:hover,
+        &:active,
+        &:focus {
+          color: $global-orange;
+        }
+
+        &.is-revealed {
+          background-image: url('#{$global-asset-path}c65b8106-icon-arrow-up.svg');
+        }
 
-      @media only screen and (min-width: 984px) {
-        width: 984px !important;
+        &::before {
+          background-image: url('#{$global-asset-path}e1b12fdd-external-link-9fa097.svg');
+          background-position: 0 3px;
+          background-repeat: no-repeat;
+          background-size: .875rem;
+          content: '';
+          height: 1.0625rem;
+          left: .625rem;
+          position: absolute;
+          width: .875rem;
+        }
       }
     }
 
     &__list {
       margin-bottom: 0;
       margin-left: 0;
       top: 0;
+
+      @media only screen and (max-width: $global-breakpoint-medium) {
+        display: none;
+
+        &.is-revealed {
+          display: block;
+
+          .global-nav__more {
+            display: block;
+          }
+        }
+      }
     }
 
     &__list-item {
-      float: left;
       display: block;
-      text-align: left;
+      float: left;
+      height: $navigaiton-height;
       margin: 0;
-      height: 30px;
       margin-top: -1px;
       position: relative;
+      text-align: left;
       top: 0;
 
+      @media only screen and (max-width: $global-breakpoint-medium) {
+        border-bottom: 1px solid $global-mid-grey;
```

Comment:
If its the last item and there is only one, it gives a little border which feels a little odd. Should we change it to be ```border-top``` instead? 
<img width="450" alt="screen shot 2017-05-03 at 10 28 15" src="https://cloud.githubusercontent.com/assets/1201618/25655996/f842d092-2fee-11e7-85ef-a694bd65fca0.png">